swap/src/fs.rs

```diff
@@ -1,4 +1,21 @@
-use std::path::Path;
+use anyhow::Context;
+use directories_next::ProjectDirs;
+use std::path::{Path, PathBuf};
+
+/// This is to store the configuration and seed files
+// Linux: /home/<user>/.config/xmr-btc-swap/
+// OSX: /Users/<user>/Library/Preferences/xmr-btc-swap/
+#[allow(dead_code)]
```

Comment:
Why is this needed?

Reply:
Because of lib and main both referencing fs (which is unnecessary). I fixed this here: https://github.com/comit-network/xmr-btc-swap/pull/166

I think putting all the PRs up was probably not a good idea, it's getting cumbersome to align things.

Reply:
#166 is merged, why is the dead code attribute still here?